### PR TITLE
Created "Browsers Profiles" extension

### PR DESCRIPTION
**Pull request by skydiver**
## Supported browsers:
* Chrome
* Chrome Canary
* Chromium
* Brave
* Microsoft Edge
* Microsoft Edge Canary
* Firefox

## Screencast
browsers-profiles-01
browsers-profiles-02

## Checklist
- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/browsers-profiles/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Raycast Browsers Profiles
+> Access your browsers profiles from Raycast
+
+More info: https://github.com/skydiver/raycast-browsers-profiles
```
Comment (mathieudutour) [inline extensions/browsers-profiles/README.md:4]: there aren't any additional information there, let's just remove it

**Review by mathieudutour — COMMENTED** [review]
Can you use `.png` instead of `.icns` for the browser icons? `.icns` aren't supported so it'd a coincidence that it works haha

And it seems like `[extensions/browsers-profiles/assets/list-icon.png](https://github.com/raycast/extensions/pull/1971/files#diff-970b3550a990e96ec6bffabb1970187270a108e4ba63f16128a81125eee3d09e)` can be removed

File: extensions/browsers-profiles/README.md (hunk 1 above)
Comment (skydiver) [reply extensions/browsers-profiles/README.md]: i posted the extension repo so i can create a guide on how to add more browsers ...

anyway, changes are ready

**Review by mathieudutour — APPROVED** [review]
Looks good :+1:

**Pull request merged**